### ISSUE TITLE
Can't reach database server at `dockerhost`

### ISSUE THREAD

**Issue by heriisei** [qoomon/docker-host#40]
I followed https://medium.com/@sam_ngu/connecting-to-docker-host-mysql-from-docker-container-linux-ubuntu-766e526542fd to connect to my host MySQL database.

Inside the other container, I could ping the dockerhost container.

My project is using Prisma as a DB access layer, and pointed to the DB via URL like this:
`mysql://user:password@dockerhost:3306/schema`.

But when I tried to access the DB. I ended up getting this error:
`"Can't reach database server at 'dockerhost':'3306'. Please make sure your database server is running at 'dockerhost':'3306'"`

All the configurations are set up as mentioned in the above medium article, like the firewall, DB user, etc.
```
dockerhost:
  image: qoomon/docker-host
  cap_add:
    - NET_ADMIN
    - NET_RAW
  restart: on-failure
  networks:
    - internal
```

The dockerhost container log is also showing no error.
```
Docker Host: 172.22.0.1 (default gateway)
Forwarding ports: 0:65535
```

I would be so happy to get any solution :smiley: from you guys.
Thank you

**Comment by qoomon** [comment]
which os do you use? you may need to bind your database port to bridge network gateway in addition to localhost(127.0.0.1).
By default most database server do bind only to 127.0.0.1

Use following docker command to get the bridge network gateway IP address
`docker network inspect bridge --format='{{( index .IPAM.Config 0).Gateway}}'`

**Comment by qoomon** [comment]
see README.md Linux Section -> https://github.com/qoomon/docker-host/blob/master/README.md#%EF%B8%8F-on-linux-systems

**Comment by heriisei** [comment]
@qoomon I'm using Linux Ubuntu. My bad, I used to write the bind-address in a wrong file (/etc/mysql/my.cnf), I just realized that my MySQL config file is /etc/mysql/mysql.conf.d/mysqld.cnf.

So I rewrite the bind-address from 127.0.0.1 to 0.0.0.0, because it didn't work with multiple IP address value configuration.
And now the connection between the container and my host database is pretty well established.

I would glad to know if there any workaround to automatically detect the dockerhost container Gateway IP address and then run a bash command to add a new firewall rule for that gateway IP address?
For now, I do manually run docker inspect <dockerhost container id> | grep Gateway to get the IP and manually run the firewall command via terminal, every time after I run docker-compose up --build.

Thank you so much.
